Vector/Model/RoomList/RecentsDataSource.m

```diff
@@ -32,10 +47,305 @@ - (instancetype)init
         
         // Replace event formatter
         self.eventFormatter = [[EventFormatter alloc] initWithMatrixSession:self.mxSession];
+        
+        favoritesCells = [[NSMutableArray alloc] init];
+        conversationCells = [[NSMutableArray alloc] init];
+        lowPriorityCells = [[NSMutableArray alloc] init];
+        
+        favoritesPos = 0;
+        conversationPos = 1;
+        lowPriorityPos = 2;
+        sectionsCount = 3;
```

Comment:
Why these values are set here?
I would expect -1 by default and sectionsCount = 0